test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1178,29 +1189,71 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			func(withCTRCfg bool) {
 				var expectedRuntime string
 				if withCTRCfg {
-					ctrcfg = newContainerRuntimeConfig(ContainerRuntimeConfigName, profile, mcp)
-					By(fmt.Sprintf("creating ContainerRuntimeConfig %q", ctrcfg.Name))
-					Expect(testclient.Client.Create(context.TODO(), ctrcfg)).ToNot(HaveOccurred(), "failed to create ctrcfg %#v", ctrcfg)
+					if hypershift.IsHypershiftCluster() {
+						ctrcfg_configmap, err = containerRuntimeConfigConfigMap(ContainerRuntimeConfigName, profile, mcp)
```

Comment:
You don't need that.
You're already get the for free by in the `Client.Create` call
https://github.com/openshift/cluster-node-tuning-operator/blob/8aa2a81399337d2cc716c210e353b82f16645d70/pkg/performanceprofile/controller/performanceprofile/hypershift/hypershift.go#L65